docs/DeveloperGuide.adoc

```diff
@@ -334,23 +379,112 @@ Use case ends.
 +
 Use case ends.
 
-* 3a. The given index is invalid.
+* 3a. The given name is invalid.
 +
 [none]
-** 3a1. AddressBook shows an error message.
+** 3a1. HackathonManager shows an error message.
 +
 Use case resumes at step 2.
 
-_{More to be added}_
+=== Use case: Find an Entity of a specific Entity Type(Participant, Mentor, Team)
+
+*MSS*
+
+1.  User requests a find an Entity of a specific Entity Type.
+2.  HackathonManager indicates success and shows the details of the Entity.
++
+Use case ends.
+
+*Extensions*
```

Comment:
Should you add an extension for invalid index? I see in C3 that you can interact with the list through both index and name.